varats/varats/tools/research_tools/vara.py

```diff
@@ -110,6 +113,11 @@ class VaRA(ResearchTool[VaRACodeBase]):
     Find the main repo online on github: https://github.com/se-passau/VaRA
     """
 
+    DEPENDENCIES = Dependencies({
+        Distro.DEBIAN: ["libboost-all-dev"],
+        Distro.ARCH: ["boost"]
+    })
```

Comment:
```suggestion
    __DEPENDENCIES = Dependencies({
        Distro.DEBIAN: ["libboost-all-dev"],
        Distro.ARCH: ["boost"]
    })
```
Should we make this private? Otherwise, users think they can change this.

Reply:
Not sure. Could be useful to others to query the dependencies of a tool though...

Reply:
That's true but then we should provide a get that copies the list. From my current understanding I only see read_only access making sense here.

Reply:
Would make sense to include that as an abstract class-level function (yes, that is a thing with python's abc) in the base class then.